docs/content/en/docs/Cluster operator/infra-api.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Deploy Cluster with Infra API"
+linkTitle: "Deploy Cluster on Infra API"
+description: >
+  The easiest way to deploy cluster with Kurator Infra API.
+---
+
+In this tutorial we’ll cover the basics of how to use [Infra API](https://github.com/kurator-dev/kurator/blob/main/pkg/apis/infra/v1alpha1/cluster_types.go) to provision Kubernetes clusters.
+
+## Prerequisites
+
+{{% readfile "includes/prerequisites.tpl" %}}
```

Comment:
another prerequisites? I remember there was one

Reply:
same one, just reuse it.

@Xieql will move this to a single page.